packages/hint/tests/lib/utils/resource-loader.ts

```diff
@@ -94,9 +94,11 @@ test('loadHint calls loadResource with the right parameters', (t) => {
 
     loadResourceStub.throws({});
 
-    t.throws(() => {
+    try {
         resourceLoader.loadHint('fake-hint');
-    });
+    } catch (e) {
```

Comment:
Is this one of the cases where we can't use `throws` because `e` is not an `Error`?

Reply:
Yes, but I think it is because the tests are wrong, it should throw an `Error`